gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigResourceLocalHandler.java

```diff
@@ -161,10 +161,14 @@ public CreateResponse createFlowConfig(FlowConfig flowConfig) throws FlowConfigL
     return this.createFlowConfig(flowConfig, true);
   }
 
+  public UpdateResponse updateFlowConfig(FlowId flowId, FlowConfig flowConfig, boolean triggerListener) {
+    // Set the max version to be the largest value so that we blindly update the flow spec in this case
+    return updateFlowConfig(flowId, flowConfig, triggerListener, Long.MAX_VALUE);
+  }
   /**
    * Update flowConfig locally and trigger all listeners iff @param triggerListener is set to true
    */
-  public UpdateResponse updateFlowConfig(FlowId flowId, FlowConfig flowConfig, boolean triggerListener) {
+  public UpdateResponse updateFlowConfig(FlowId flowId, FlowConfig flowConfig, boolean triggerListener, long modifiedWatermark) {
```

Comment:
Same thing in this PR, just update the name to avoid confusion. Basically we will not update the entry if the modification time is larger than the "modifiedWatermark" / "version"